src/distributed/topology_leader.ts

```diff
@@ -203,53 +207,77 @@ export class TopologyLeader {
                     if (!perform_loop || alive_workers.length == 0) {
```

Comment:
line 194: if (alive_workers.length == 0)
Why do we handle dead worksers if at least one is live and not handle otherwise?

Reply:
I agree. fixed.